apps/meteor/server/modules/listeners/listeners.module.ts

```diff
@@ -209,6 +209,10 @@ export class ListenersModule {
 		});
 
 		service.onEvent('watch.inquiries', async ({ clientAction, inquiry, diff }): Promise<void> => {
+			if (settings.get('Livechat_Routing_Method') !== 'Manual_Selection') {
+				return;
+			}
+
```

Comment:
Let's keep this change to another PR, looks unrelated to the purpose of this PR. In the next PR we can explain better what is the purpose of not publishing inquiries if the routing method is equal to 'Manual_Selection'.

Reply:
Hmmm, refactor is for the whole queue. And sending inquiries to clients are also part of the queue.

Not sending them when the algo is not manual selection is a perf improvement for the streaming of inquiries, since when the algo is not manual, we don't even show them on the client.
When the algorithm changes, UI fetches all the inquiries again so the cached collection in the UI is emptied. So all of these events are just sent for nothing.